src/Utilities/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -7,7 +7,7 @@ Design: CA1000-CA1099
 Globalization: CA1300-CA1399, CA2101
 Mobility: CA1600-CA1699
 Performance: CA1800-CA1899
-Security: CA2100-CA2199, CA2300-CA2399, CA3147
+Security: CA2100-CA3199,
```

Comment:
>CA2100-CA3199 [](start = 10, length = 13)

This will cause overlaps as Usage rules below already use `CA2200-CA2299` :-)

Reply:
Whoops, maybe I need an analyzer for the other analyzer's input... :-)